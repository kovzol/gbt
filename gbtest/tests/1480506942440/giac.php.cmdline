#!/bin/sh
cd tests/1480506942440
cat giac.php.in | /usr/bin/timeout -k 120 60 /usr/bin/time /usr/bin/giac > giac.php.out 2>giac.php.err