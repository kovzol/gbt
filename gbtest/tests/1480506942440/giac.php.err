// Using locale /usr/share/locale/
// C
// /usr/share/locale/
// giac
// UTF-8
// Maximum number of parallel threads 8
Help file /usr/share/giac/doc/local/aide_cas not found
Added 0 synonyms

// Groebner basis computation time 0.002023 Memory 0.024804M
// Time 0
0.04user 0.00system 0:00.05elapsed 96%CPU (0avgtext+0avgdata 24804maxresident)k
0inputs+16outputs (0major+1593minor)pagefaults 0swaps
