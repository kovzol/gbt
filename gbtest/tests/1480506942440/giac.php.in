gbasis([x^2+x*y-1,2*x*y+3*x,x^3],[x,y],plex)