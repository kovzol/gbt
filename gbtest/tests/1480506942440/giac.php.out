Welcome to giac readline interface
(c) 2001,2016 B. Parisse & others
Homepage http://www-fourier.ujf-grenoble.fr/~parisse/giac.html
Released under the GPL license 3.0 or above
See http://www.gnu.org for license details
May contain BSD licensed software parts (lapack, atlas, tinymt)
-------------------------------------------------
Press CTRL and D simultaneously to finish session
Type ?commandname for help
0>> gbasis([x^2+x*y-1,2*x*y+3*x,x^3],[x,y],plex)
[1]
1>> 